ios-swiftui/realm-template-apps-swiftui/Views/LogoutButton.swift

```diff
@@ -0,0 +1,22 @@
+import SwiftUI
+import RealmSwift
+
+/// Logout from the synchronized realm. Returns the user to the login/sign up screen.
+struct LogoutButton: View {
+    @State var isLoggingOut = false
+    
+    var body: some View {
+        Button("Log Out") {
+            guard let user = realmApp.currentUser else {
+                return
+            }
+            isLoggingOut = true
+            user.logOut() { error in
```

Comment:
Is there an async/await version of this now?

Reply:
Yup - good catch. I've swapped it out to use the async/await version, but the error handling is a bit tricky for the Logout button. I can pass the error to the Error View or display it as Text, but it just displays in the nav bar and the button goes away. I'm inclined to think an error here should display as an alert the user can dismiss, but I need to do a little spelunking to refresh myself on how to do that.